config/opendatahub-io/org.yaml

```diff
@@ -113,6 +113,7 @@ orgs:
     - VannTen
     - vconzola
     - VedantMahabaleshwarkar
+    - vmudadla
```

Comment:
@VaniHaripriya this should be your gh username so: `VaniHaripriya` same with the other one